dev-support/create-release/release-util.sh

```diff
@@ -378,8 +378,8 @@ make_binary_release() {
 
   # Check there is a bin gz output. The build may not produce one: e.g. hbase-thirdparty.
   f_bin_tgz="./${PROJECT}-assembly/target/${basename}*-bin.tar.gz"
-  if test -f "${f_bin_tgz}"; then
-    cp "${f_bin_tgz}" ..
+  if ls ${f_bin_tgz} &>/dev/null; then
```

Comment:
nit: this is also true if ${f_bin_tgz} is a directory. I think a stronger check is [[ -f ${f_bin_tgz}]] (very similar to what it was before) and you can get rid of /dev/null